packages/core/admin/admin/src/utils/getFetchClient.js

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+import { auth } from '@strapi/helper-plugin';
+
+const instance = axios.create({
+  baseURL: process.env.STRAPI_ADMIN_BACKEND_URL,
```

Comment:
Could you add a test case for this line?

Reply:
What do we want to test here exactly?

Reply:
we have tried to test it, let me know if it is ok, thanks